### PR TITLE
BJMIR-632 Add tournament details to match live feed snapshot

### PR DESCRIPTION
**Pull request by mkingori**

### CODE REVIEW

**Review by melias122 — APPROVED** [review]

**Comment by melias122** [comment]
Before merge, wait for 2 approval

**Comment by mkingori** [comment]
@pmachan please review this PR.

File: proto/bragi/cs2.proto
Comment (kafkapre) [inline proto/bragi/cs2.proto:45]: `Tournament tournament = 7;` should be enough.

Occurs multiple times

Comment (mkingori) [reply proto/bragi/cs2.proto]: Fixed

**Review by kafkapre — APPROVED** [review]

**Comment by melias122** [comment]
Appved @mkingori I think it will need another regenerate

**Comment by mkingori** [comment]
> Appved @mkingori I think it will need another regenerate

There were no new changes that needed regenerating

**Pull request merged**
